.env.example

```diff
@@ -21,6 +21,7 @@ VITE_FEATURE_MULTISET=true
 
 APP_NAME=RetroAchievements
 APP_ENV=local
+VITE_BASE_URL=localhost
```

Comment:
Shouldn't this be down with the other VITE settings?

https://github.com/RetroAchievements/RAWeb/blob/416f2bdfa1280e1bd227e3b4afae814c03849b6a/.env.example#L124

Reply:
Forgot that was down there. Updated in the branch and on the server.